src/snippets/muiCard.ts

```diff
@@ -0,0 +1,13 @@
+import snip from '../snip'
+
+export const description = 'Material-UI <Card>'
+export const body = snip`
+<Card>
```

Comment:
I thought about making a card snippet, the only thing is that not all cards need to have `CardContent` and `CardActions`, some have a `CardHeader`, etc.  Let me look back through all the different card examples and think about it.  I'm thinking we should make a few different card snippets for common cases, but the tricky question is what to name the various snippets.